src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -230,31 +230,56 @@ public Response getDataset(@Context ContainerRequestContext crc, @PathParam("id"
             return ok(jsonbuilder.add("latestVersion", (latest != null) ? json(latest, true) : null));
         }, getRequestUser(crc));
     }
-    
-    // This API call should, ideally, call findUserOrDie() and the GetDatasetCommand 
-    // to obtain the dataset that we are trying to export - which would handle
-    // Auth in the process... For now, Auth isn't necessary - since export ONLY 
-    // WORKS on published datasets, which are open to the world. -- L.A. 4.5
+
     @GET
+    @AuthRequired
     @Path("/export")
     @Produces({"application/xml", "application/json", "application/html", "application/ld+json", "*/*" })
-    public Response exportDataset(@QueryParam("persistentId") String persistentId, @QueryParam("exporter") String exporter, @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response) {
+    public Response exportDataset(@Context ContainerRequestContext crc, @QueryParam("persistentId") String persistentId,
+            @QueryParam("version") String versionId, @QueryParam("exporter") String exporter,
+            @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response) {
 
         try {
             Dataset dataset = datasetService.findByGlobalId(persistentId);
             if (dataset == null) {
                 return error(Response.Status.NOT_FOUND, "A dataset with the persistentId " + persistentId + " could not be found.");
             }
-            
+
+            DataverseRequest req = createDataverseRequest(getRequestUser(crc));
+            DatasetVersion datasetVersion = null;
+            try {
+                String versionToLookUp = DS_VERSION_LATEST_PUBLISHED;
+                if (versionId != null) {
+                    versionToLookUp = versionId;
+                }
+                datasetVersion = getDatasetVersionOrDie(req, versionToLookUp, dataset, uriInfo, headers);
+            } catch (WrappedResponse wr) {
+                // wr.getLocalizedMessage() is null so don't bother returning it
+                return error(BAD_REQUEST, "Unable to look up dataset based on version. Try " + DS_VERSION_LATEST_PUBLISHED + " or " + DS_VERSION_DRAFT + ".");
+            }
+
+            if (!datasetVersion.isDraft()) {
+                Command<DatasetVersion> cmd = new GetLatestPublishedDatasetVersionCommand(dvRequestService.getDataverseRequest(), dataset);
+                DatasetVersion latestPublishedVersion = commandEngine.submit(cmd);
```

Comment:
If  `versionId` was null, we should have already looked up the latest published version by now - so, we could skip looking it up again here, I think - ?

Reply:
Yes, good catch. Fixed in 8776fdbfd5.